components/user_settings/import_theme_modal.tsx

```diff
@@ -59,26 +66,26 @@ export default class ImportThemeModal extends React.Component {
         const colors = text.split(',');
         const theme = {type: 'custom'};
 
-        theme.sidebarBg = colors[0];
-        theme.sidebarText = colors[5];
-        theme.sidebarUnreadText = colors[5];
-        theme.sidebarTextHoverBg = colors[4];
-        theme.sidebarTextActiveBorder = colors[2];
-        theme.sidebarTextActiveColor = colors[3];
-        theme.sidebarHeaderBg = colors[1];
-        theme.sidebarHeaderTextColor = colors[5];
-        theme.onlineIndicator = colors[6];
-        theme.mentionBg = colors[7];
-        setThemeDefaults(theme);
-
-        this.state.callback(theme);
+        (theme as any).sidebarBg = colors[0];
```

Comment:
We have a `Theme` type you can use instead, it's in `mattermost-redux`.

Reply:
`import {Theme} from 'mattermost-redux'`
gave me error: `Module '"../../node_modules/mattermost-redux"' has no exported member 'Theme'.`
what was I wrong here?

Reply:
You need to define the path that the `Theme` type is in.
It should look like:
`import {Theme} from 'mattermost-redux/types/preferences'`

Reply:
thank you!